doc/adr/0011-tsp-analysis-api.md

```diff
@@ -178,100 +183,175 @@ sequenceDiagram
     client->>client: Refresh UI
 ```
 
-### Configuration service per experiment
+### Configure customizable outputs 
+
+Outputs might accept input parameters that will configure the analysis used to create the output. Such customization can create new outputs or provide additional query parameters. The TSP api will allow the back-end to define configuration types for that. Each configuration type will use `CustomizationTypeSource` to define what parameters need to be provided to create a new and derived output. The derived output descriptor will store the configuration details. Derived outputs can be deleted using a DELETE operation.
 
-For this data provider service will be augmented for managing configurations per experiment.
+    GET /experiments/{expUUID}/outputs/{outputId}/configTypes
```

Comment:
Are these independent from the global configuration types defined above? If you create a global configuration instance, can it be used to create a derived output? Although, in global configuration is there anything to indicate to which output the configuration can apply?

Reply:
They are independent. These configuration instances will only be for the given data provider. They live until the corresponding data provider is deleted or the experiment is deleted. 

Sharing these configurations across experiments through the global configuration is out of scope of this update. We can consider it in the future when required.